chapters/3-measurement.tex

```diff
@@ -37,125 +39,209 @@
+This is because, if you ever need to perform a record linkage such as a fuzzy match
+on string variables like proper names, 
+you will make fewer errors the more information you have.
+If you ever need to do a fuzzy match,
+you should always do that between the master dataset
+and the dataset without an unambiguous identifier.
+You should not do anything with that dataset until
+you have successfully merged
+the project IDs from the master dataset.
+
+With the master datasets as an authoritative source 
+of the project ID and all research variables,
+it serves as an unambiguous method of mapping
+the observations in your study to your research design.
+
+The third and final component of the data plan is the \textbf{data flow charts}.
+Each analysis dataset
+(see Chapter 6 for discussion on why you likely need multiple analysis datasets)
+should have a data flow chart that is a diagram 
+where each starting point is either a master dataset 
+or a dataset listed in the data linkage table.
+The data flow chart should include instructions on how 
+all datasets should be combined to create the analysis dataset.
+The operations used to combine the data could, for example, be
+appending, one-to-one merging, 
+many-to-one or one-to-many merging, collapsing, or a broad variety of others.
+You must list which ID variable or set of ID variables
+should be used in these operations,
+and if the operation creates a new variable or combination of variables
+that identifies the newly linked data.
+Once you have acquired the datasets listed in the flow chart, 
+you can also start listing the number of observations that both
+the starting point dataset should start with and
+the combined datasets should have after each operation. 
+This is a great method to track attrition and to make sure that
+the operations used to combine datasets did not create unwanted duplicates
+or incorrectly drop any observations.
+
+A data flow chart can be created in a flow chart drawing tool
+(there are many free alternatives online),
+by using shapes in Microsoft PowerPoint,
+or simply by drawing on a piece of paper and taking a photo.
+However, we recommend that the charts are created in a digital tool
+so that new versions can easily be created 
+once you learn new things about your research throughout your project.
+
+\subsection{Defining research variables related to your research design}
+
+After you have set up your data plan,
+you need to carefully think about your research design 
+and which research variables you will need in the data analysis
+to infer the relation between differences in measurement variables 
+and your research design.
+We assume you have a working familiarity
+with the research designs mentioned here.
+If needed, you can reference \textcolor{red}{Appendix XYZ},
+where you will find more details 
+and specific references for common impact evaluation methods.
+
 The research designs discussed here compare a group that received
```

Comment:
Suggest: As DIME primarily works on impact evaluations, we focus our discussion here on research designs that compare a group that received ... (continue the sentence as is)

Reply:
done 27dee583f8c60b28fee2d31d9275c6b7d82b02af